packages/s/sqlite3/package.yml

```diff
@@ -19,15 +24,17 @@ setup      : |
     if [[ -z "${EMUL32BUILD}" ]]; then
         ex_opts="--enable-readline"
     fi
+    %patch -p1 -i $pkgfiles/sqlite-lemon-system-template.patch
 
     %configure --disable-static \
-        --enable-dynamic-extensions \
         --enable-fts3 \
         --enable-fts4 \
         --enable-fts5 \
         --enable-rtree \
         --enable-threadsafe \
         $ex_opts \
+        --disable-tcl \
+        --soname=legacy \
```

Comment:
Similar comment: I would not add new flags unless they fix something broken.

Reply:
I assume `--soname=legacy` is to preserve the soname version, but I don't really understand why you'd need `--disable-tcl`. Does `--soname=legacy` require `--disable=tcl`?